src/lib/pubsub/pubsub_publish.h

```diff
@@ -5,6 +5,7 @@
 /* See LICENSE for licensing information */
 
 #ifndef TOR_PUBSUB_PUBLISH_H
+#define TOR_PUBSUB_PUBLISH_H
```

Comment:
Do we have a tool to check whether inclusion guards in our headers are correct? If not, maybe we should find/make one?

Reply:
I've opened a new ticket (https://trac.torproject.org/projects/tor/ticket/29542 ) to do this.

Reply:
This seems to actually be https://trac.torproject.org/projects/tor/ticket/29756

Reply:
Ah yes; that's correct.